programs/sbf/rust/account_mem/src/lib.rs

```diff
@@ -108,7 +108,20 @@ pub fn process_instruction(
             // memmov dst overlaps begin of account
             unsafe { sol_memmove(too_early(3).as_mut_ptr(), buf.as_ptr(), 10) };
         }
-
+        14 => {
+            // memmove dst overlaps begin of account, reverse order
```

Comment:
my bad for not reviewing the master PR, but shouldn't these tests all test:

- success in the boundary condition: dst=too_early(0), src=too_early(0)
- failure at boundary-1: dst=too_early(0), src=too_early(1)
- failure at boundary-N: dst=too_early(0), src=too_early(N)

Reply:
I'll add tests on master

Reply:
See https://github.com/anza-xyz/agave/pull/4650